packages/react-core/src/components/Tabs/__tests__/Tabs.test.tsx

```diff
@@ -492,3 +492,29 @@ test('should not render scroll buttons when isVertical is true', () => {
   expect(screen.queryByLabelText('Scroll left')).not.toBeInTheDocument();
   expect(screen.queryByLabelText('Scroll right')).not.toBeInTheDocument();
 });
+
+test('should render a disabled add button', () => {
+  render(
+    <Tabs onAdd={jest.fn()} addButtonAriaLabel="add-label" isAddButtonDisabled={true}>
+      <Tab eventKey={0} title={<TabTitleText>"Tab item 1"</TabTitleText>} closeButtonAriaLabel="close-label">
+        Tab 1 section
+      </Tab>
+    </Tabs>
+  );
+
+  const addButton = screen.getByLabelText('add-label');
+  expect(addButton.hasAttribute('disabled')).toBe(true);
```

Comment:
For these 2 new tests, let's use the `toBeDisabled` assertion instead of checking whether the disabled attribute is truthy/falsy:

```suggestion
  expect(addButton).toBeDisabled()
```

Then for the opposite assertion just `expect(addButton).not.toBeDisabled()`

If we want to add a separate prop for aria-disabled, then we'd want to assert against the attribute having a specific value, though.